### DISCUSSION TITLE
Can extension be checked installed?

### DISCUSSION THREAD

**Discussion by Future-Outlier** [coder/code-server#6552] [Q&A] [answered]
Hello, I am a contributor from Flyte.
We are developing a plugin using `code-server`, and to utilize it, we need to use `code-server --install-extension` to install some extensions.
We are wondering if there's a way to check the extension is installed?
So that we can avoid installing it twice, thanks a lot!

**Comment by code-asher** [comment, marked as answer]
I believe `--install-extension` skips installation if it is already installed.  Although, it might query the marketplace to see if there is an update.

You could do `--list-extensions` and see if the ID is in the list.
